vendor.mod

```diff
@@ -98,6 +98,8 @@ require (
 	resenje.org/singleflight v0.3.0
 )
 
+require github.com/container-orchestrated-devices/container-device-interface v0.5.4
```

Comment:
This seems oddly-placed, it probably should be part of the block above. I think `go mod tidy` will take care of that for you.

Reply:
I ran `./hack/vendor.sh tidy` and `./hack/vendor.sh vendor` and commited the changes to `vendor.*` and `vendor`. If there is another recommended way to do this, I can do that instead.

Reply:
I will manually move the dependency and re-run the scripts.

Reply:
Manually moving and running `./hack/vendor.sh tidy` seems to have cleaned things up.